src/test/regress/sql/bfv_joins.sql

```diff
@@ -604,6 +603,10 @@ from
       ttt.tc1=ttt1.tc2
 ) t on true;
 
+drop table ttt;
+drop table ttt1;
+
+-- error when use motion deliver a lateral param
```

Comment:
Why did you remove this line?

Reply:
'-- error when use motion deliver a lateral param' was removed because queries from the test now doesn't produce any error